db/migrate/20160323171249_remove_deleted_from_attachments.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDeletedFromAttachments < ActiveRecord::Migration
+  def change
+    remove_column :attachments, :deleted, :boolean, null: false, default: false
```

Comment:
I've force-pushed this branch without this migration and the schema change. These changes are available in https://github.com/alphagov/whitehall/tree/remove-soft-delete